### ISSUE TITLE
[Bug Report] Open/remaining v-select a11y violations

### ISSUE THREAD

**Issue by yg-paytm** [vuetifyjs/vuetify#8733]
### Environment
**Vuetify Version:** 2.0.10
**Vue Version:** 2.6.10
**Browsers:** Firefox 68.0
**OS:** Mac OS 10.14

### Steps to reproduce
- Create a v-select component
- Download Axe a11y scanner extension for Chrome/Firefox
- Run Axe analysis on the rendered page

### Expected Behavior
All framework components should pass a11y checks from scanner apps like Axe.

### Actual Behavior
V-select component in its current implementation is violating ARIA attribute rules.

![Image of Axe Analysis](https://i.imgur.com/kpnJQfJ.png)

### Reproduction Link
<a href="https://codepen.io/frobear91/pen/BaBWGGK?&editable=true&editors=101" target="_blank">https://codepen.io/frobear91/pen/BaBWGGK?&editable=true&editors=101</a>

**Comment by KaelWD** [comment]
Fixed by #10226
